build.gradle

```diff
@@ -204,6 +204,18 @@ dependencies {
     // library to build pdf reports
     compile 'com.github.dhorions:boxable'
 
+    //support for Oracle DB version 12.1.0.2
```

Comment:
can you put a comment to explain why it is commented out, and how it can be used (credentials)?